frontend/src/themes/colors.ts

```diff
@@ -1,5 +1,6 @@
 const colors = {
   blue: {
+    25: "#F7F9FB",
```

Comment:
I don't love the fact that we're adding a new colour here. Is there any other colour it looks similar to (I seem to remember some blueish greys already)? If not, fine, but I feel like for visual cohesiveness we should only add if necessary.